userdocs/profiles.dev/docs/installer-docs/setting-values.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 2
+---
+
+# Configuring values
+
+Users can partially configure their profile installations with the `--config-map`
+flag on the `pctl install` command.
+
+## Helm Chart artifact values
+
+To configure Helm Chart artifacts in a profile, first create a [ConfigMap](https://kubernetes.io/docs/concepts/configuration/configmap/).
+
+For example, to configure a profile with the following artifacts:
+
+```yaml
+# ...
+spec:
+  # ...
+  artifacts:
+  - name: fluent-bit
+    chart:
+      path: "fluent-bit/chart"
+  - name: nginx-server
+    chart:
+      url: https://charts.bitnami.com/bitnami
+      name: nginx
+      version: "8.9.1"
+  # ...
```

Comment:
shall we add some bits to the profile author docs about this? Two things stick out:
1. Plz name your artifacts sensibly so when I configure them its easy to remember the name
2. Perhaps provide an example configmap that does some example configuration?

Reply:
yeh i think that is sensible